tests/IndexRangeAccess/IndexRangeAccess.sol

```diff
@@ -6,4 +6,4 @@ contract Demo {
     function log(bytes calldata value) external {
         emit Log(bytes(value[0:10]));
     }
-}
\ No newline at end of file
+}
```

Comment:
hehe these files are supposed to be not very well formatted anyway so I wouldn't care much about the line at the end. but would make sure the snapshot fixes this.

Reply:
sorry it's my bad I can't help but fix a missing a new line at EOF 😊